packages/phoenix-event-display/src/managers/ui-manager/phoenix-menu/phoenix-menu-node.ts

```diff
@@ -163,6 +163,8 @@ export class PhoenixMenuNode {
         value: config?.['value'],
         highValue: config?.['highValue'],
       });
+      config.setEnableMin?.(config?.['enableMin']);
```

Comment:
I think this may be what is causing the behavior. You can try removing this just to see if it works. What may happen is that the UI in Angular rerenders and reconstructs the node in the tree and calls these functions.

Reply:
No, I don't think so: I commented it out and no change (and I'm not sure why it would only affect the min checkbox and not the max?)

I fear it's something I did wrong in the html (which is what I'm weakest on!) - i.e. the checkbox is somehow part of the slider (or vice versa). But I can't see it if so...